lib/srv/desktop/windows_server.go

```diff
@@ -351,6 +364,8 @@ func NewWindowsService(cfg WindowsServiceConfig) (*WindowsService, error) {
 		s.cfg.Log.Infoln("desktop discovery via LDAP is disabled, set 'base_dn' to enable")
 	}
 
+	s.ca = windows.NewCertificateAuthority(s.clusterName, s.cfg.LDAPConfig, s.lc, s.cfg.AccessPoint, s.cfg.Log)
```

Comment:
nit: What would you cay about renaming `NewCertificateAuthority` to something like `NewStoreClient` ? 

At first glance `ca` and  `NewCertificateAuthority` naming is a bit misleading for a reader and indicatate the ca contains some kind of `CertificateAuthority` object but actually the ca is a windows store client that allows to update ca in that store.

Reply:
I think that's a good idea.